Quadriga/src/main/java/edu/asu/spring/quadriga/service/network/domain/impl/TextOccurance.java

```diff
@@ -113,6 +116,39 @@ public boolean equals(Object obj) {
                 return false;
         } else if (!textUri.equals(other.textUri))
             return false;
+        if (author == null) {
+            if (other.author != null)
+                return false;
+        } else if (!author.equals(other.author))
+            return false;
+        if (title == null) {
+            if (other.title != null)
+                return false;
+        } else if (!title.equals(other.title))
+            return false;
+        if (creationDate == null) {
+            if (other.creationDate != null)
+                return false;
+        } else if (!creationDate.equals(other.creationDate))
+            return false;
         return true;
     }
+    public String getAuthor() {
+        return author;
+    }
+    public void setAuthor(String author) {
+        this.author = author;
+    }
+    public String getTitle() {
+        return title;
+    }
+    public void setTitle(String title) {
+        this.title = title;
+    }
+    public String getCreationDate() {
+        return creationDate;
+    }
+    public void setCreationDate(String creationDate) {
+        this.creationDate = creationDate;
+    }
```

Comment:
changed the place of getters and setters for the author, title and creation date(i.e., written after the other getters and setters)